commune_app/tests/test_Expense_model.py

```diff
@@ -0,0 +1,33 @@
+from ..models import Expense, User
+import pytest
+
+
+class TestExpenseModel:
+    @pytest.mark.django_db
+    def test_new_expense(self):
+        expense = Expense(title="buying some food", budget="150", date="2011-09-01T13:20:30+03:00", assign=User(first_name="Faez", last_name="Zangariya", email="faezzangariyacs@gmail.com", phone_number="+972523324193"))
+        assert expense.title == "buying some food"
+        assert expense.budget == "150"
+        assert expense.date == "2011-09-01T13:20:30+03:00"
+        assert expense.assign.first_name == "Faez"
+
+    @pytest.mark.django_db
+    def test_create_expense(self):
+        expense = Expense.create_expense(title="Laundry", budget="200", date="2011-09-01T13:20:30+03:00", assign=User.create_user(first_name="Ron", last_name="Tur", email="rontur@gmail.com", phone_number="+972523324193"))
+        assert expense.title == "Laundry"
+        assert expense.budget == "200"
+        assert expense.date == "2011-09-01T13:20:30+03:00"
+        assert expense.assign.first_name == "Ron"
+
+    @pytest.mark.django_db
+    def test_get_expense_by_title(self):
```

Comment:
same as above 
you could use the same user you already created as a fixture like:
```suggestion
    def test_get_expense_by_title(self, user1):
```